xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -851,9 +852,35 @@ bool CGUIMediaWindow::Update(const std::string &strDirectory, bool updateFilterP
     }
   }
 
-  // if we haven't found the selected item, select the first item
+  // if we haven't found the selected item, see if we should select the first unplayed item
+  // if yes, find the first unplayed item and select it
+  // if no, select the first item
   if (!bSelectedFound)
-    m_viewControl.SetSelectedItem(0);
+  {
+    int iIndex = 0; // index of the item to select, default to the first item
+
+    // Check if we should select the first unplayed item
```

Comment:
You can tell I'm not a big music listener, I hadn't even though of that.

I can easily change it to JumpToFirstUnplayedSeasonOrEpisode (or something less wordy) if required.